src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -245,8 +245,8 @@ public bool AddTimeData(IPAddress peerAddress, TimeSpan offsetSample, bool isInb
             if (startWarningLoopNow)
                 this.StartWarningLoop();
 
-            this.logger.LogTrace("(-):{0}", res);
-            return res;
+            this.logger.LogTrace("(-):{0}", success);
+            return success;
         }
```

Comment:
Your changes in this method are not improving the code objectively. Please avoid changes that are subjective. Otherwise another coder will later revert this change back for his own subjective reasons and then you will revert the reversion ... There is no code style rule that would prescribe not using `res` for a result value.

Reply:
Are you telling me that you have a policy that actually allows writing a shortened word like that?

result isn't even appropriate here either, so I changed it because I thought result is the wrong word and also because shortening words is a very bad practice - wideley accepted by the industry.

a boolean of res is not right and needs improving. Do you have a better suggestion than success?  it looked like it was indicating to be whether it was successful or not, hence success.  But certainly not res.

Reply:
we don't have policy against `res` and i would prefer `res` over `success`, i also disagree that shortening is very bad practice

we differ in our opinions on what should be used here, we can talk about that on skype, but since we are at the level of opinions, it should not be changed

Reply:
happy to discuss that on skype, not here

Reply:
ok that is fine, I will revert that one for now - I agree if it is opinion based that we shouldn't changed. It is just unusual for me to hear an opinion of res being ok. That said it has happened now and I will undo.

Reply:
This is what I like about you, you are willing to accept something you don't agree with at least until we have time to discuss it properly